tests/test_multiprocessing.py

```diff
@@ -1,9 +1,11 @@
+from __future__ import print_function
+
 import os
 import math
 import time
 import pickle
 import pprint
-import random
+from random import random, choice
```

Comment:
No strong opinion, only that `random.random` causes a small lookup penalty, and since it is used in a tight loop, it was just a performance/readability thing. 
I also prefer to keep track of imported things with the usual import mechanism, but sometimes I will break that rule for really common stuff (like `from datetime import datetime`), or for performance reasons. Your call here :wink: